packages/documentation/gatsby-config.js

```diff
@@ -38,8 +38,17 @@ module.exports = {
         start_url: `/`,
       },
     },
+    `gatsby-plugin-sass`,
```

Comment:
Was the order of the plugins breaking anything?

Reply:
The order of the plugins shouldn't break things. I re ordered the `gatsby-plugin-mdx` because I was testing another plugin that had to be after the `gatsby-source-filesystem` and I never changed the order back. Most examples have the `gatsby-source-filesystem` before the mdx plugin but I don't think it's necessary. i can always revert the order to confirm too.

Reply:
If the order doesn't matter, and the current order reflects the order of common examples, then I don't think it's necessary to revert the order.